library/globals.inc.php

```diff
@@ -288,6 +288,8 @@ function gblTimeZones()
             array(
                 'dual' => xl('Dual'),
                 'comprehensive' => xl('Comprehensive'),
+                'fixed' => xl('fixed'),
+                'none' => xl('none'),
             ),
             'dual', // default
             xl('Search Patient By Any Demographics, Dual additionally lets direct access to Patient Finder, Comprehensive has collapsed input box')
```

Comment:
Provide a quick explanation to the Fixed and None setting in line above.

Reply:
sure